src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyPageAction.java

```diff
@@ -15,11 +15,19 @@
 
     @Override
     protected ActionResult execute() throws EntityDoesNotExistException {
+        String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID);
+        Assumption.assertNotNull(courseId);
+
+        String feedbackSessionName = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        Assumption.assertNotNull(feedbackSessionName);
+
         List<InstructorAttributes> instructors = logic.getInstructorsForGoogleId(account.googleId);
         Assumption.assertNotNull(instructors);
         
         InstructorFeedbackEditCopyPageData data = new InstructorFeedbackEditCopyPageData(account);
         data.courses = new ArrayList<CourseAttributes>();
+        data.courseId = courseId;
+        data.fsName = feedbackSessionName;
```

Comment:
Use set methods instead of accessing attributes directly, in case we declare them as private later.